configure.ac

```diff
@@ -109,10 +109,9 @@ AS_IF([test "x$SYSCONFSTATICDIR" = x],
       [sysconfstaticdir=$SYSCONFSTATICDIR])
 AC_SUBST([sysconfstaticdir])
 
-
+AC_PROG_CC
 AM_PROG_CC_C_O
 AC_PROG_MKDIR_P
-AC_PROG_CC_STDC
 AC_PROG_YACC
 AC_CANONICAL_HOST
 AC_C_CONST
```

Comment:
Does it mean that these AC_* macros are not necessary? Does it automatically check for bison/flex? (AC_PROG_YACC)?

Reply:
This removal is related to following release note item:

    *** AC_PROG_CC_STDC, AC_PROG_CC_C89, AC_PROG_CC_C99 are now obsolete.
    
      Applications should use AC_PROG_CC.

Reply:
OK, thanks for info.